components/html-block/html-block.js

```diff
@@ -118,6 +118,12 @@ const getRenderers = () => {
  */
 class HtmlBlock extends LitElement {
 
+	static get properties() {
+		return {
+			_noDeferredRendering: { type: Boolean, attribute: 'no-deferred-rendering', reflect: true }
```

Comment:
So... `type` was typo'd as `Type` and Lit was ignoring the extra property and (correctly) assuming it should use its default serialization (which is `type: String`). Fixing the typo makes this behave like a boolean attribute as expected.

Reply:
Is this designated at "private" because we only want the LMS to use it?

Reply:
So right now this isn't what the component uses to determine whether rendering should be deferred. Instead, the component just sets this internally so styling can be applied properly. I was trying to keep to the original pattern of "if it's a template, defer; otherwise don't", with the additional caveat that the consumer needs to provide a specific container `div` for us to actually render.

Reply:
So basically as written now, nothing should set this externally because the html-block is just going to overwrite it anyways.